index.js

```diff
@@ -39,9 +39,70 @@ const columns = [
     width: "120px",
   },
 ];
+const toolbar = [
+  {
+    type: "i",
+    content: "undo",
+  },
+  {
+    type: "i",
+    content: "redo",
+  },
+  {
+    type: "i",
+    content: "save",
+  },
+  {
+    type: "i",
+    content: "format_bold",
+    k: "font-weight",
+    v: "bold",
+  },
+  {
+    type: "i",
+    content: "format_italic",
+  },
+  {
+    type: "color",
+    content: "format_color_text",
+    k: "color",
+  },
+  {
+    type: "color",
+    content: "format_color_fill",
+    k: "background-color",
+  },
+  {
+    type: "i",
+    content: "format_underlined",
+  },
+  {
+    type: "i",
+    content: "strikethrough_s",
+  },
+  {
+    type: "i",
+    content: "border_all",
+  },
+  {
+    type: "i",
+    content: "format_align_center",
+    k: "text-align",
+    v: "center",
+  },
+  {
+    type: "i",
+    content: "vertical_align_bottom",
+  },
+  {
+    type: "i",
+    content: "format_text_wrap",
+  },
+];
 new Lightsheet(document.getElementById("lightsheet"), {
   data,
   columns,
+  toolbar,
   onCellChange: (colIndex, rowIndex, newValue) => {
     console.log(colIndex, rowIndex, newValue);
   },
```

Comment:
The user will also provide a boolean of toolbar visibility, and the dom(optional) he wants the toolbar to mound into.
example,

const lightsheet = new Lightsheet(document.getElementById("lightsheet"), {
  data,
  columns,
 {
showToolbar: true/false,
items: toolbar,
element: document.getElementById("toolbar-dom-id")
},
  onCellChange: (colIndex, rowIndex, newValue) => {
    console.log(colIndex, rowIndex, newValue);
  },
}
//if user wants to make the toolbar visible/imvisible, he can call this method anytime
lightsheet.showToolbar(true/false);

Reply:
You can name your methods and property names as you see fit